x/execution/internal/types/msg.proto

```diff
@@ -76,4 +71,14 @@ message MsgUpdate {
     // error is an error result.
     string error = 4;
   }
+
+  // Start time
+  uint64 start = 5 [
+    (gogoproto.moretags) = 'validate:"required"'
+  ];
+  
+  // Stop time
+  uint64 stop = 6 [
+    (gogoproto.moretags) = 'validate:"required,gtfield=Start"'
+  ];
```

Comment:
Let's use a better type here or at least say what is it in the comment.
You can use `gogoproto.casttype` to cast it to `time.Duration` or something a bit better than `uint64`.

Reply:
Is it even a good idea to use `uint64` type in the proto?
You used a string for the task' prices.
I would recommend using a safe type in proto, and a specific one in Go using `gogoproto.casttype` or `gogoproto.customtype`

Reply:
Duration is different than time but I did replace the time from `uint64` to `int64` to avoid many conversions.